aten/src/ATen/functorch/BatchRulesLinearAlgebra.cpp

```diff
@@ -485,6 +485,49 @@ pinv_batch_rule(
     const std::optional<int64_t> rtol_bdim, bool hermitian) {
   return atol_rtol_tensor_batch_rule(ATEN_FN2(linalg_pinv, atol_rtol_tensor), input, input_bdim, atol, atol_bdim, rtol, rtol_bdim, hermitian, "linalg.pinv");
 }
+
```

Comment:
Can you add tests? It's suspicious that none of the xfails in test_vmap flipped

Reply:
Yeah the SDPA tests are skipped because of randomness.